api/proxy/server/handlers_cert.go

```diff
@@ -95,10 +95,9 @@ func (svr *Server) handleGetShared(
 	// to decode the payload themselves inside the fpvm.
 	returnEncodedPayload := parseReturnEncodedPayloadQueryParam(r)
 
-	payloadOrEncodedPayload, err := svr.sm.Get(
+	payloadOrEncodedPayload, err := svr.certMgr.Get(
 		r.Context(),
 		versionedCert,
-		mode,
```

Comment:
You've remove the actual usage of `mode` here, but it's still being passed in and logged below, *as if* it is being used.

Consider removing param entirely, or renaming to `_` to preserve the method signature

Reply:
[67796f8](https://github.com/Layr-Labs/eigenda/pull/1893/commits/67796f893a40dcd93817b975eec21fc33f43800e)